package.json

```diff
@@ -55,6 +59,7 @@
     "jsdom": "16.2.2",
     "mini-css-extract-plugin": "^0.9.0",
     "node-sass": "4.14.1",
+    "np": "https://github.com/dopecodez/np.git#54f3c8f",
```

Comment:
Note to self: we can probably also move back to the stable version. Turns out the version name we wanted to have isn't https://semver.org/ compatible anyways 😉